src/main/webapp/navbar.jsp

```diff
@@ -0,0 +1,11 @@
+<nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+    <a href="/activity">Activity</a>
```

Comment:
It looks like this doesn't contain links to Marcel's admin page. It would be good to include that logic so it could be reached from everywhere as well.

Reply:
Yup, I agree.